src/subscript/fmu_copy_revision/fmu_copy_revision.py

```diff
@@ -140,7 +144,6 @@
 - ert/output/
 - ert/storage/
 - ert/**/storage/
-- rms/output
```

Comment:
Could the documentation line further up more clearly state what it means to have an entry in DIRFILTER<int> ?

Reply:
Yes, certainly needed. When doing that more errors and imprecise wording were revealed, so there will come additional changes